vita3k/packages/src/pkg.cpp

```diff
@@ -271,9 +271,25 @@ bool install_pkg(const fs::path &pkg_path, EmuEnvState &emuenv, std::string &p_z
         LOG_INFO(string_name);
 
         if ((byte_swap(entry.type) & 0xFF) == 4 || (byte_swap(entry.type) & 0xFF) == 18) { // Directory
-            fs::create_directories(path / string_name);
+            if (!fs::create_directories(path / string_name)) {
+                LOG_ERROR("Failed to create directory");
+                evp_cleanup();
+                return false;
```

Comment:
fs::create_directories will return false if the directory already exists, we may want to be a bit careful with this (for update I wouldn't be surprised if this happens).

Reply:
Good point. There's an overload of the function that gives an error code that can be checked instead.